woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2433,32 +2406,16 @@ def test_accessor_types(sample_df):
     assert returned_types.shape[1] == 3
     assert len(returned_types.index) == len(sample_df.columns)
 
-    correct_physical_types = {
-        'id': Integer.primary_dtype,
-        'full_name': Unknown.primary_dtype,
-        'email': EmailAddress.primary_dtype,
-        'phone_number': Unknown.primary_dtype,
-        'age': IntegerNullable.primary_dtype,
-        'signup_date': Datetime.primary_dtype,
-        'is_registered': BooleanNullable.primary_dtype,
-    }
+    correct_physical_types = {name: ltype.primary_dtype for name, ltype in sample_inferred_logical_types.items()}
+    if ks and isinstance(sample_df, ks.DataFrame):
```

Comment:
We are trying to get away from this pattern with the new helper functions. You can use `if _is_koalas_dataframe(sample_df):` here instead.